robot/EDA/tests/browser/settings/eda_settings_course_connections_edit_mode.robot

```diff
@@ -7,22 +7,30 @@ Library         cumulusci.robotframework.PageObjects
 Suite Setup     Open Test Browser
 Suite Teardown  Capture screenshot and delete records and close browser
 
+Test Setup      Run keywords
+...             Go to EDA settings tab          Course Connections      AND
+...             Update enable cc to default
+
 *** Test Cases ***
 Validate Edit Mode For Course Connections, Settings
-    Go to EDA settings tab                  Course Connections
-
-    #Go into Edit Mode
-    Click Button                            Edit
-
-    #Verify that the checkbox can be modified.  
-    Wait for Locator                        eda_settings.cc_checkbox
-    Click on Element                        eda_settings.cc_checkbox
-
-    ${student_select} =  Get Eda Locator    eda_settings.student_select
-    ${faculty_select} =  Get Eda Locator    eda_settings.faculty_select
-
-    Select From List By Label       ${student_select}   Student
-    Select From List By Label       ${faculty_select}   Faculty
-
-    #Save settings
-    Click Button                                Save
+    [tags]                                      unstable        W-041783
+    Click edit on EDA settings page
+    Verify enable course connections warning    true
+    Set enable course connections
+    Verify enable course connections warning    false
+    Verify dropdown values                      Default Active Student Record Type
+    ...                                         Default
+    ...                                         Faculty
+    ...                                         Student
+    Verify dropdown values                      Default Faculty Record Type
+    ...                                         Default
+    ...                                         Faculty
+    ...                                         Student
```

Comment:
is it possible to modify or enhance the functionality of verify dropdown values to read  multiple lists ? Then you don't need to call the same keyword twice ?

Reply:
What do you mean by that? Like having sub-dictionaries?

Reply:
yes, it is not a must but something to consider in the future